test/unit/org/apache/cassandra/index/sai/cql/types/multicell/FrozenTupleCollectionTest.java

```diff
@@ -31,7 +31,7 @@ public class FrozenTupleCollectionTest extends IndexingTypeSupport
     @Parameterized.Parameters(name = "dataset={0},wide={1},scenario={2}")
     public static Collection<Object[]> generateParameters()
     {
-        return generateLiteralParameters(new FrozenTupleDataSet(
+        return generateParameters(new FrozenTupleDataSet(
```

Comment:
nit: I just noticed that these `Frozen*` tests are in the `multicell` package. Not a big deal, but odd :D